activerecord/test/models/man.rb

```diff
@@ -2,6 +2,7 @@
 
 class Man < ActiveRecord::Base
   has_one :face, inverse_of: :man
+  has_one :autosave_face, class_name: "Face", autosave: true, foreign_key: :autosave_man_id, inverse_of: :autosave_man
```

Comment:
```suggestion
  has_one :autosave_face, class_name: "Face", autosave: true, foreign_key: :man_id, inverse_of: :autosave_man
```

(This will be `human_id` after a rebase.)